tox.ini

```diff
@@ -101,6 +104,15 @@ deps =
 commands =
     bandit -c {toxinidir}/pyproject.toml -r {[vars]src_path} {[vars]tst_path}
 
+[testenv:interface]
```

Comment:
Do you know whether we run this target in GH workflow?

Reply:
We run all the default targets, that is, the ones defined as `envlist` at the top of the `tox.ini` file